moped-editor/src/views/projects/projectView/ProjectComponents/CreateComponentModal.js

```diff
@@ -76,7 +76,9 @@ const CreateComponentModal = ({
     }
   };
 
-  const onClose = () => {
+  const onClose = (event, reason) => {
+    if (reason && reason === "backdropClick")
```

Comment:
cool to see how this is implemented, and I like that this retains the ability to press `ESC` to exit the modal! I think that, between the esc and the close button (which can be tabbed into focus or clicked), there are plenty of ways to exit the form with intention.

Reply:
+1 on the escape -- it was the first thing that I tried when I was experimenting with it. Works great.